codex_of_the_damned/navigation.py

```diff
@@ -138,6 +138,22 @@ def walk(self, path=None, top=None, ante=None, post=None):
                 Nav(lazy_gettext("Weenie AUS")),
                 Nav(lazy_gettext("Weenie DEM")),
                 Nav(lazy_gettext("Weenie DOM")),
+                Nav(
+                    lazy_gettext("Bibliodèque"),
+                    index=True,
+                    children=[
```

Comment:
Needing confirmation about positioning into navigation helper

Reply:
Seems like the right spot to me !